### PR TITLE
add support of event.screenX and event.screenY properties (closes #2325)

### LINKED ISSUE #2325: Touch events are not working while page is blocked?
### Are you requesting a feature or reporting a bug?

I'm not sure if this is a missing feature, or actually a bug...

### What is the current behavior?

I've been able to simulate touch gestures using `hammer-touchemulator` here: https://www.boxfactura.com/pulltorefresh.js/demos/trigger.html 

With that enabled the demo page is also working on desktop browsers.

### What is the expected behavior?

I expect my tests run with this page the same way.

### How would you reproduce the current behavior (if this is a bug)?

First try to _move_ the trigger to enable the PTR, it'll work.

Now, run the test below to _drag_ the same trigger.

Also I noticed that running the same test with `-d` enabled, and then unlocking the page, allowed me to pull the trigger as usual...

#### Provide the test code and the tested page URL (if applicable)

Tested page URL: https://www.boxfactura.com/pulltorefresh.js/demos/trigger.html 

Test code

```js
fixture('Trigger test')
  .page('https://www.boxfactura.com/pulltorefresh.js/demos/trigger.html');

test('try to drag', async t => {
  await t
    .drag('.trigger', 0, 100, { speed: 0.1 })
    .drag('.trigger', 0, 200, { speed: 0.1 });
});

```

I'm doing wrong or there's a preferred way to perform this kind of test?

### Specify your

* operating system: OSX 10.13.3
* testcafe version: 0.19.2
* node.js version: v9.9.0

### PR DESCRIPTION
**Pull request by AlexKamaev**

### CODE REVIEW

File: src/client/automation/playback/move/index.js [hunk 1]
```diff
@@ -271,8 +271,12 @@ export default class MoveAutomation {
     _emulateEvents (currentElement) {
         var whichButton = this.holdLeftButton ? eventUtils.WHICH_PARAMETER.leftButton : eventUtils.WHICH_PARAMETER.noButton;
         var button      = this.holdLeftButton ? eventUtils.BUTTONS_PARAMETER.leftButton : eventUtils.BUTTONS_PARAMETER.noButton;
+        var screenLeft  = window.screenLeft ? window.screenLeft : window.screenX;
```
Comment (helen-dikareva) [inline src/client/automation/playback/move/index.js]: `window.screenLeft  ||  window.screenX;`

Comment (AlexKamaev) [reply src/client/automation/playback/move/index.js]: omg

File: test/functional/fixtures/regression/gh-2325/test.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+describe('[Regression](GH-2325)', function () {
```
Comment (AndreyBelym) [inline test/functional/fixtures/regression/gh-2325/test.js]: You can remove the functional test since you've refactored it into the client test.

**Comment by AlexKamaev** [comment]
fixed \cc @helen-dikareva @AndreyBelym

**Pull request merged**